### PR TITLE
Add notice for "createdAt" key in API

### PR DESCRIPTION
**Pull request by Kakulukian**
For https://github.com/huggingface/moon-landing/pull/7991 (private)

### CODE REVIEW

File: docs/hub/api.md [hunk 1]
```diff
@@ -142,11 +150,20 @@ params = {
...
 
 This is equivalent to `huggingface_hub.space_info(repo_id, revision)`.
 
+The key `createdAt` in the response begins on `2022-03-02T23:29:04.000Z` as a result of a special migration.
+
+### GET /api/metrics
+
+Get information from all metrics in the Hub.
+
+This is equivalent to `huggingface_hub.list_metrics()`.
```
Comment (julien-c) [inline docs/hub/api.md:166]: stale addition

Comment (julien-c) [reply docs/hub/api.md:166]: (see PR diff)

File: docs/hub/api.md [hunk 2]
```diff
@@ -46,12 +46,16 @@ params = {
 
 This is equivalent to `huggingface_hub.list_models()`.
 
+The key `createdAt` in the response begins on `2022-03-02T23:29:04.000Z` as a result of a special migration.
```
Comment (julien-c) [inline docs/hub/api.md:49]: IMO this should be mentioned just once, under a `# Repo creation times` heading, with a wording like:
> For all repos-returning API calls, the `createdAt` attribute is the time when the repo was created.
> Note that there is a special value of `2022-03-02T23:29:04.000Z` for all repos that were created before we started storing the creation date.

cc'ing @Wauplin too

**Review by julien-c — CHANGES_REQUESTED** [review]

File: docs/hub/api.md (hunk 2 above)
Comment (Kakulukian) [reply docs/hub/api.md:49]: Yeah sure

**Review by julien-c — APPROVED** [review]
lgtm but wdyt @Wauplin?

**Review by Wauplin — APPROVED** [review]
Looks good! Will integrate it to `huggingface_hub` as well :fire:

**Comment by julien-c** [comment]
looks fine in prod IMO, thanks:

image

**Pull request merged**